### PR TITLE
Close #14484: Make the Heartline Twister ratings less hateful

### LINKED ISSUE #14484: Make the Heartline Twister ratings less hateful
While some roller coaster types are inevitably more popular, or comfortable, or _fun_ than others, and their ratings must reflect that, I think the ratings for the Heartline Twister are really bonkers. Marcel Vos asked himself if an Ultra Twister caused Chris Sawyer a lot of childhood trauma.

The original Ultra Twister was very popular. There seemed to be no complaints about things like comfort. Despite this, the Heartline Twister gets _penalised_ for having air time - the only roller coaster type to get it, seemingly for no good reason.

I think we should change the ratings to be more in line with other steel coaster - it is both more realistic and balanced. The lack of versatility will of course remain, so I doubt we will have to fear they'll suddenly take over the RCT2 landscape ;-)

### PR DESCRIPTION
**Pull request by Gymnasiast**
This raises two parameters: the base excitement value (which is very low, especially since the air-time bonus is limited too) and the bonus value (the novelty of this ride type warrants a bigger value).

It leaves the rest of the poor bonuses as-is, as some of them (like scenery and synchronisation bonuses) might just be there because the track obscures the view.

### CODE REVIEW

File: src/openrct2/network/NetworkBase.cpp [hunk 1]
```diff
@@ -42,7 +42,7 @@
 // This string specifies which version of network stream current build uses.
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
-#define NETWORK_STREAM_VERSION "17"
+#define NETWORK_STREAM_VERSION "18"
```
Comment (LeftofZen) [inline src/openrct2/network/NetworkBase.cpp:45]: why does network stream version need to be changed?

Comment (ZehMatt) [reply src/openrct2/network/NetworkBase.cpp:45]: Because it will change the outcome of the simulation

**Comment by duncanspumpkin** [comment]
Should this be changed?

**Comment by CookiePLMonster** [comment]
Slightly offtopic, I find it a bit funny that this got PR'd exactly the next day after Marcel Vos made a video on this coaster and said "I feel like OpenRCT2 should **not** change this 😅

**Comment by spudpiggy** [comment]
It should at least be an option for those that still want the memetic terribleness of base heartline.

**Comment by duncanspumpkin** [comment]
> It should at least be an option for those that still want the memetic terribleness of base heartline.

No that just leads to complexity. I don't know of a single person that uses the legacy ratings code if anything that reinforces the fact there should be only one.

**Comment by spudpiggy** [comment]
> > It should at least be an option for those that still want the memetic terribleness of base heartline.
> 
> No that just leads to complexity. I don't know of a single person that uses the legacy ratings code if anything that reinforces the fact there should be only one.

fair

**Comment by Gymnasiast** [comment]
I put the original behind the ORIGINAL_RATINGS define, so people **can** compile OpenRCT2 with these original, terrible ratings if they want to. Although, like Duncan pointed out, I doubt many people do.

**Comment by spudpiggy** [comment]
> I put the original behind the ORIGINAL_RATINGS define, so people **can** compile OpenRCT2 with these original, terrible ratings if they want to. Although, like Duncan pointed out, I doubt many people do.

ok xd

**Comment by ZehMatt** [comment]
> I put the original behind the ORIGINAL_RATINGS define, so people **can** compile OpenRCT2 with these original, terrible ratings if they want to. Although, like Duncan pointed out, I doubt many people do.

I would actually get rid of that entirely, I dislike secret configurations that aren't part of the build system out of the box and should be only put in place if absolutely required.

**Comment by CookiePLMonster** [comment]
> I would actually get rid of that entirely, I dislike secret configurations that aren't part of the build system out of the box and should be only put in place if absolutely required.

I have to disagree, I think it works well for documentation purposes. It's a clear indicator of "we changed The Original Vision:tm: here"

**Comment by ZehMatt** [comment]
> > I would actually get rid of that entirely, I dislike secret configurations that aren't part of the build system out of the box and should be only put in place if absolutely required.
> 
> I have to disagree, I think it works well for documentation purposes. It's a clear indicator of "we changed The Original Vision™️ here"

Use good old comments to document code and not configurations.

**Comment by Gymnasiast** [comment]
We use the `ORIGINAL_RATINGS` define in more places, and unless we decide to cut all of them, one should be included here. The decision on whether or not to cut them is outside the scope of this PR and should be carried out on Discord or in a GitHub issue.

Now, let’s get back to the changes themselves. ;-)

**Pull request merged**